src/mono/mono/metadata/native-library.c

```diff
@@ -1240,8 +1298,17 @@ lookup_pinvoke_call_impl (MonoMethod *method, MonoLookupPInvokeStatus *status_ou
 #ifndef HOST_WIN32
 retry_with_libcoreclr:
 #endif
-	// FIXME: these flags are not getting passed correctly
-	module = netcore_lookup_native_library (alc, image, new_scope, 0);
+	cinfo = mono_custom_attrs_from_method_checked (method, error);
+	flags = get_dllimportsearchpath_flags (cinfo);
+
+	if (flags < 0) {
+		cinfo = mono_custom_attrs_from_module (m_class_get_image (method->klass), error);
+		flags = get_dllimportsearchpath_flags (cinfo);
+	}
+	if (flags < 0)
+		flags = 0;
+	printf ("Here's some flags: %d!!!\n", flags);
```

Comment:
debugging output
```suggestion
```

Reply:
I'll remove these at the end, I will convert this to a draft PR